README.md

```diff
@@ -1,21 +1,18 @@
 # ING Open Banking SDK
 With this SDK we provide the means to generate drivers to interact with ING's Open Banking APIs. It includes a custom generator that creates drivers which encapsulate HTTP signature signing and OAuth application token flows. As shown by the demo-app these drivers significantly reduce complexity to interact with ING's Open Banking APIs. Since the drivers are generated from OpenAPI Specification files, updating is easy by regenerating the drivers based on updated files.
```

Comment:
Can you please review the last line here 

"Since the drivers are generated from OpenAPI Specification files, updating is easy by regenerating the drivers based on updated files."

Its difficult to understand this line.